ReverseString.java

```diff
@@ -0,0 +1,18 @@
+public class ReverseString {
+    public static void main(String[] args) {
+        {
+            String testString = "Have a nice day my friend";
+            char[] result = ReverseString(testString);
+            System.out.println(result);
+            //ReverseString(testString);
```

Comment:
delete all not necessary comment before pushing

Reply:
f526ad89